resultsdbupdater/utils.py

```diff
@@ -182,66 +182,67 @@ def ci_metrics_post_to_resultsdb(msg):
 
 def cips_post_to_resultsdb(msg):
     session = retry_session()
+
     # define variables from cips.json
     msg_id = msg['headers']['message-id']
-    ci_type = msg['headers']['ci_type']
+    type = msg['headers']['type']
     component = msg['headers']['component']
-    brew_task_id = msg['headers']['brew_task_id']
+    id = msg['headers']['id']
+    scratch = msg['headers']['scratch']
     msg_body = msg['body']['msg']
-    tests = msg_body['tests']
-
-    arch = msg_body['environment']['arch']
-    brew_tag = msg_body['environment']['brew_tag']
-    build_type = msg_body['environment']['build_type']
-
-    jenkins_job_url = msg_body['infrastructure']['jenkins_job_url']
-    jenkins_build_url = msg_body['infrastructure']['jenkins_build_url']
-
-    cips_report = msg_body['results']['cips_report']
-    cips_status = msg_body['results']['cips_status']
-
-    testcase_url = jenkins_job_url
+    msg_body_ci = msg['body']['msg']['ci']
+    msg_body_artifact = msg['body']['msg']['artifact']
+    url = msg_body['run']['url']
+    outcome = msg_body['status']
+    category = msg_body['category']
+    issuer = msg_body_artifact['issuer']
+    rebuild = msg_body['run']['rebuild']
+    log = msg_body['run']['log']
+    system_os = msg_body['system'][0]['os']
+    system_provider = msg_body['system'][0]['provider']
+    ci_name = msg_body_ci['name']
```

Comment:
A lot of these variables are only used once, so rather than assigning these values to a variable, can you just use them directly when assigning the value of `result_data`? I think that's easier to read.